src/main/java/duke/Parser.java

```diff
@@ -0,0 +1,15 @@
+package duke;
+
+/**
+ * Class that parse the input from the user's input.
+ */
+public class Parser {
+    String input;
+    public Parser(String input) {
+        this.input = input;
+    }
+
+    public void parse() {
+
+    }
```

Comment:
What is the purpose of this method? Perhaps methods with no definition should be in abstract classes.